src/addons/transitions/ReactCSSTransitionGroup.js

```diff
@@ -26,14 +26,15 @@ function createTransitionTimeoutPropValidator(transitionType) {
       // If no timeout duration is provided
       if (props[timeoutPropName] == null) {
         return new Error(
-          timeoutPropName + ' wasn\'t supplied to ReactCSSTransitionGroup: ' +
-          'this can cause unreliable animations and won\'t be supported in ' +
-          'a future version of React. See ' +
-          'https://fb.me/react-animation-transition-group-timeout for more ' +
-          'information.'
+          timeoutPropName +
+            " wasn't supplied to ReactCSSTransitionGroup: " +
```

Comment:
You may be interested in the implementation of the string printing.
https://github.com/prettier/prettier/blob/master/src/printer.js#L2836-L2909